src/scripts/App.js

```diff
@@ -0,0 +1,30 @@
+import ModuleManifest from "./ModuleManifest";
+
+export default class App {
+  static defaults = {
+    moduleAttribute: "data-module",
+    optionsAttribute: "data-module-options"
+  };
+
+  constructor(scope = document, config = {}) {
+    this.config = { ...config, ...App.defaults };
```

Comment:
```suggestion
    this.config = { ...App.defaults, ...config };
```

Right? Config values override our defaults?

Reply:
Oops. That was copypasta from Brian's changes.

Reply:
Updated.